stylesheets/_mixin.responsive-utilities.scss

```diff
@@ -91,3 +91,31 @@
         @include sr-hide;
     }
 }
+
+@mixin word-wrap() {
+    -ms-word-break: break-all; // ie11
+    -ms-hyphens: auto;
+    -moz-hyphens: auto;
+    -webkit-hyphens: auto;
+    word-break: break-all; // edge
+    overflow-wrap: break-word; // various old FF and Saf
+    word-wrap: break-word; // various old FF and Saf
+    word-break: break-word; // chrome
+    hyphens: auto; // not supported in chrome
+
+    // trigger word breaks and hyphens in FF
+    @-moz-document url-prefix() {
+        word-break: normal;
+    }
+}
+
+@mixin visuallyhidden {
```

Comment:
You've duplicated an existing mixin called `sr-hide`, the timeline code should be updated to use this

Reply:
Updated as advised